### DISCUSSION TITLE
Neorg no longer renders text correctly

### DISCUSSION THREAD

**Discussion by Tvrt1e** [nvim-neorg/neorg#1373] [Q&A]
I've been using neorg for about a month now, but all of a sudden neorg stopped working. :Neorg works fine, but in .norg files no Neorg features work. I installed neorg using vim-plug, so I removed it and installed via rocks.nvim, but still nothing worked. Any ideas as to why? Maybe I misunderstood the configuration
neorg.lua:
```
require('neorg').setup {
load = {
    ["core.defaults"] = {
        config = {
            disable = {
                -- module list goes here
                "core.autocommands",
                "core.itero",
            },
        },
    },
}
}
```
Reverting to 7.0.0 fixes this, but I would rather use the latest version if possible.

**Comment by pysan3** [comment]
@Tvrt1e @curelesss 

- https://vhyrro.github.io/posts/neorg-and-luarocks/
- https://github.com/nvim-neorg/neorg/issues/1342
- https://github.com/pysan3/Norg-Tutorial/blob/main/MIGRATION-v8.md

**Reply by Tvrt1e** [reply]
Thanks for the reply. I'm using rocks.nvim, should I switch to lazyvim? I've tried all of the steps in those documents that don't apply to lazyvim.

**Reply by curelesss** [reply]
Thank you for pointing me to the right place for troubleshooting.

Manage to get Neorg 8.0 install correctly by running `:Lazy build luarocks.nvim` & `:Lazy build neorg`

btw, Neovim outputs of building `Luarocks` states it requries `unzip` .

**Reply by pysan3** [reply]
@Tvrt1e AFAIK rocks.nvim is still beta release so you have to dig into the source code to fix your issue, and you are on your own. 

Rocks.nvim and lazy.nvim are incompatible so if you are already using lazy, you cannot use rocks for installation. 

If you are using some other pm, there is no feature to automatically run `build.lua` so you'll need to specify it yourself. Read https://github.com/nvim-neorg/neorg/issues/1342#issuecomment-2020696535

PM other than lazy should be compatible with rocks tho so trying to debug rocks would also be a good way to go.

**Reply by Tvrt1e** [reply]
I switched to lazy.nvim and got the exact same behavior as with Rocks.nvim. I'm starting to think maybe something other than neorg is the problem, but to my kowledge neorg is the only thing that changed. Everything in Neorg works except for anything that changes the quality of text. Also vimwiki works perfectly fine.

**Comment by pysan3** [comment]
@Tvrt1e Give me your config including neorg's lazy spec as well as luarocks.nvim.

**Reply by Tvrt1e** [reply]
init.vim:
```
lua << EOF
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not (vim.uv or vim.loop).fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "https://github.com/folke/lazy.nvim.git",
    "--branch=stable", -- latest stable release
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)
require("lazy").setup("plugins")
EOF
```


neorg.lua:
```
return {
   {
	"nvim-treesitter/nvim-treesitter", 
	build = ":TSUpdate"
   },

    {
        "vhyrro/luarocks.nvim",
        priority = 1000, -- We'd like this plugin to load first out of the rest
        config = true, -- This automatically runs `require("luarocks-nvim").setup()`
    },
    {
        "nvim-neorg/neorg",
        dependencies = { "luarocks.nvim", "nvim-treesitter" },
	version = "*",
        config = function()
            require("neorg").setup({
            load = {
        ["core.defaults"] = {},
        ["core.concealer"] = {},
	},
	    })
        end,
    }
}
```

**Reply by pysan3** [reply]
What do you mean by none of the neorg features work?

**Reply by Tvrt1e** [reply]
All neorg features work save for the ones that modify the way text looks. :Neorg command works, and headers render with concealer enabled.
![image](https://github.com/nvim-neorg/neorg/assets/78752428/7cd0870c-9ac6-40d9-bc14-84c7c1fa9376)

**Reply by pysan3** [reply]
So what does *not* work?

**Reply by Tvrt1e** [reply]
It really just comes down to colors. Text looks like plain text as opposed to colored/highlighted text, etc..

**Reply by Tvrt1e** [reply]
I know that bold/colored text functions work in neovim using my colorscheme because code syntax is properly colored and all vimwiki features work without a hitch.

**Reply by pysan3** [reply]
I found you were having the same discussion here so let me join there. 

- https://github.com/nvim-neorg/neorg/issues/1374
